cluster/join_test.go

```diff
@@ -35,7 +35,7 @@ func Test_SingleJoinOK(t *testing.T) {
 
 	defer ts.Close()
 
-	j, err := Join([]string{ts.URL}, "id0", "127.0.0.1:9090", false, nil,
+	j, err := Join("", []string{ts.URL}, "id0", "127.0.0.1:9090", false, nil,
```

Comment:
Let's add a unit test where we actually pass in the local addr, and not just "".